### PR TITLE
KAA-1612: [Reviewed] Using C++ SDK, Raspberry Pi. [master]

### PR DESCRIPTION
**Pull request by hunch7**

### CODE REVIEW

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-RPi/index.md [hunk 1]
```diff
@@ -10,113 +10,141 @@ sort_idx: 20
...
+   cd $RPI_ROOT
+   wget -O boost_1_59_0.tar.bz2 http://sourceforge.net/projects/boost/files/boost/1.59.0/boost_1_59_0.tar.bz2/download
+   tar -xvf boost_1_59_0.tar.bz2
+   cd boost_1_59_0
+   ./bootstrap.sh
+   sed -r 's/(using\s+gcc)(\s+;)/\1 : arm : arm-linux-gnueabihf-c++\2/g' -i project-config.jam
+   ./bjam install toolset=gcc-arm --prefix=${INSTALLDIR}
+   ```
+
+5. Install [Apache Avro](https://avro.apache.org/) for the host and target machnies.
```
Comment (Polinaa) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-RPi/index.md]: machines

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-RPi/index.md [hunk 2]
```diff
@@ -10,113 +10,141 @@ sort_idx: 20
...
+   tar -xvzf Botan-1.11.27.tgz
+   cd Botan-1.11.27
+   python configure.py --cpu=arm --cc-bin=${CROSS}-g++ --prefix=${INSTALLDIR}
+   make && make install
+   cp -r ${INSTALLDIR}/include/botan-1.11/botan/ ${INSTALLDIR}/include
+   rm -r ${INSTALLDIR}/include/botan-1.11
+   ```
+
+7. Download Kaa C++ SDK from your [Kaa instance]({{root_url}}Glossary/#kaa-instance-kaa-deployment) and compile it.
+See [Generate SDK](Programming-guide/Your-first-Kaa-application/#generate-sdk).
```
Comment (Polinaa) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-RPi/index.md]: Broken link: page not found

**Review by Polinaa — APPROVED** [review]

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-RPi/index.md (hunk 1 above)
Comment (hunch7) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-RPi/index.md]: Fixed.

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-RPi/index.md (hunk 2 above)
Comment (hunch7) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-RPi/index.md]: Fixed.

**Review by vadimol — APPROVED** [review]

**Pull request merged**